File: forloop-add-test/forloop-add-test.c
// gcc -Wall -O2 teszt1.c -o teszt1

#include <stdio.h>
#include <stdlib.h>

int main(int argc, char *argv[]) {
    long toval = atoll(argv[1]);
    long i, j, sumval = 0;
    for (i=0; i<toval; i++) {
        for (j=0; j<toval; j++) {
            sumval += i+(j^1);
        }
    }
    printf("C: %lu\n", sumval);
}

File: forloop-add-test/forloop_add_test.php
#!/usr/bin/env php
<?php
function main($toval) {
    $sumval = 0;
    for ($i=0; $i < $toval; $i++) {
        for ($j=0; $j < $toval; $j++) {
            $sumval += $i+($j^1);
        }
    }
    echo 'PHP: ', $sumval;
}

main( intval($argv[1]) )
?>
